.github/workflows/pull-request-test.yml

```diff
@@ -149,12 +149,13 @@ jobs:
             useSession,
             checkWorkflows,
             rstudioSession,
+            v2/anonymousNavigation,
             v2/dashboardV2,
-            v2/projectBasics,
             v2/groupBasics,
+            v2/projectBasics,
             v2/projectResources,
             v2/searchEntities,
-            v2/anonymousNavigation
+            v2/sessionBasics,
```

Comment:
Nitpick: why is this not just `v2/sessions`?

Reply:
I followed a similar pattern to the names I gave to the other files. I thought adding other tests for the same target (E.G. sessionWithCredentials) would avoid overlapping names, but it isn't very important.
I can simplify it, but I would rather change all the other names too to use a similar pattern everywhere (groupBasics --> groups, ...)